README.adoc

```diff
@@ -309,24 +400,94 @@ The default host name for Docker Desktop is `localhost`.
 The default host name for minikube is {minikube-ip}. Otherwise it can be found using the `minikube ip` command.
 --
 
-Then `curl` or visit the following URLs to access your microservices, substituting the appropriate host name:
+Then use `curl` or visit the following URLs to access your microservices, substituting the appropriate host name:
 
-- `{system-api}`
-- `{inventory-api}/system-service`
+* `{system-api}`
+* `{inventory-api}/system-service`
 
 The first URL returns system properties and the name of the pod in an HTTP header called `X-Pod-Name`.
 To view the header, you may use the `-I` option in the `curl` when making a request to `{system-api}`.
-The second URL adds properties from `system-service` to the inventory
-{kube} Service. Visiting `{inventory-api}/[kube-service]` in general adds to the inventory
-depending on whether `kube-service` is a valid {kube} Service that can be accessed.
+The second URL adds properties from `system-service` to the inventory {kube} Service. 
```

Comment:
The second URL adds properties from `system-service` to the inventory {kube} Service. 
--->
The second URL adds properties from `system-service` endpoint to the inventory {kube} service.